src/edit.js

```diff
@@ -36,47 +53,67 @@ export default function Edit( { attributes, setAttributes } ) {
 	useEffect( () => {
 		async function getRelatedPosts() {
 			const postId = window.location.href.match( /post=([\d]+)/ )[ 1 ];
-			const response = await window.fetch( `/wp-json/wp/v2/posts?search=${ countries[ countryCode ] }&exclude=${ postId }` );
+			const response = await window.fetch(
+				`/wp-json/wp/v2/posts?filter[s]=${ countries[ countryCode ] }&exclude=${ postId }`
+			);
 
 			if ( ! response.ok )
 				throw new Error( `HTTP error! Status: ${ response.status }` );
 
 			const posts = await response.json();
 
 			setAttributes( {
-				relatedPosts: posts?.map( ( relatedPost ) => ( {
-					...relatedPost,
-					title: relatedPost.title?.rendered || relatedPost.link,
-					excerpt: relatedPost.excerpt?.rendered || '',
-				} ) ) || [],
+				relatedPosts:
+					posts?.map( ( relatedPost ) => ( {
+						...relatedPost,
+						title: relatedPost.title?.rendered || relatedPost.link,
+						excerpt: relatedPost.excerpt?.rendered || '',
+					} ) ) || [],
 			} );
 		}
 
 		getRelatedPosts();
-	}, [countryCode, setAttributes] );
+	}, [ countryCode, setAttributes ] );
 
 	return (
-		<>
+		<div { ...blockProps }>
 			<BlockControls>
 				<ToolbarGroup>
-						<ToolbarButton label={ __( 'Change Country', 'xwp-country-card' ) }
-							icon={ edit } onClick={ handleChangeCountry } disabled={ ! Boolean( countryCode ) } />
+					<ToolbarButton
+						label={ __( 'Change Country', 'xwp-country-card' ) }
+						icon={ edit }
+						onClick={ handleChangeCountry }
+						disabled={ ! Boolean( countryCode ) }
+					/>
 				</ToolbarGroup>
 			</BlockControls>
 			<div>
-				{ isPreview ? <Preview countryCode={ countryCode } relatedPosts={ relatedPosts }/> : <Placeholder icon={ globe } label={ __( 'XWP Country Card', 'xwp-country-card' ) }
-								 isColumnLayout={ true }
-								 instructions={ __( 'Type in a name of a contry you want to display on you site.', 'xwp-country-card' ) }>
+				{ isPreview ? (
+					<Preview
+						countryCode={ countryCode }
+						relatedPosts={ relatedPosts }
+					/>
+				) : (
+					<Placeholder
+						icon={ globe }
+						label={ __( 'XWP Country Card', 'xwp-country-card' ) }
+						isColumnLayout={ true }
+						instructions={ __(
+							'Type in a name of a contry you want to display on you site.',
```

Comment:
There are typos in the label:

```suggestion
							'Type in a name of a country you want to display on your site.',
```